lib/helper.js

```diff
@@ -30,16 +31,127 @@ let logofied = text => {
   );
 };
 
+let tablerForArray = data => {
+  if (!Array.isArray(data)) {
+    return "";
+  }
+  let finalTable = new Table({
+    chars: {
+      top: "═",
+      "top-mid": "╤",
+      "top-left": "╔",
+      "top-right": "╗",
+      bottom: "═",
+      "bottom-mid": "╧",
+      "bottom-left": "╚",
+      "bottom-right": "╝",
+      left: "║",
+      "left-mid": "╟",
+      mid: "─",
+      "mid-mid": "┼",
+      right: "║",
+      "right-mid": "╢",
+      middle: "│"
+    }
+  });
+
+  let firstItem = data[0];
+  let head = ["#"];
+  if (typeof firstItem == "object") {
+    for (prop in firstItem) head.push(prop);
+    finalTable.push(head);
+    // making the body
+    data.forEach((item, index) => {
+      let temp = [index];
+      for (prop in item) {
+        temp.push(item[prop]);
+      }
+      finalTable.push(temp);
+    });
+  } else {
+    head.push("");
+    finalTable.push(head);
+    data.forEach((item, index) => {
+      finalTable.push([index, item]);
+    });
+  }
+  // console.log(finalTable);
+  return finalTable.toString();
+};
+
+let tablerForObject = data => {
+  let finalTable = new Table({
+    chars: {
+      top: "═",
+      "top-mid": "╤",
+      "top-left": "╔",
+      "top-right": "╗",
+      bottom: "═",
+      "bottom-mid": "╧",
+      "bottom-left": "╚",
+      "bottom-right": "╝",
+      left: "║",
+      "left-mid": "╟",
+      mid: "─",
+      "mid-mid": "┼",
+      right: "║",
+      "right-mid": "╢",
+      middle: "│"
+    }
+  });
+  let head = [];
+  let body = [];
+
+  // making the head of the table
+  for (prop in data) head.push(prop);
+  finalTable.push(head);
+
+  // making the body of the table
+  for (prop in data) {
+    if (typeof data[prop] == "object") {
+      if (Array.isArray(data[prop])) body.push(tablerForArray(data[prop]));
+      else {
+        body.push(tablerForObject(data[prop]));
+      }
+    } else {
+      body.push(data[prop]);
+    }
+  }
+  finalTable.push(body);
+  return finalTable.toString();
+};
+
+let tabler = data => {
+  if (Array.isArray(data)) {
+    return tablerForArray(data);
+  } else if (typeof data == "object") {
+    return tablerForObject(data);
+  } else {
+    console.log(
```

Comment:
You don't need to show this log when the data can not be represented in a Table structure